UnitedIncome/components
# ActionBar Component

### Description
> Please provide a brief description of your changes below.

Adds the `ActionBar` component. This mostly just abstracts the existing one from McG, but also adds the ability to have a back button. 

### Testing Instructions
> What should be done in order to test this pull request, or re-create its effects.

* It should behave the same in McG.
* It's generic enough to be used on top of whatever screen needs it.

### Screenshots
> If this pull request includes a visual change please attach any applicable screenshots.

### Comments
> Any additional comments that will aid with the review process.

- [JIRA Ticket](https://unitedincome.atlassian.net/browse/app-*)
